### ISSUE TITLE
Fixated interest with abnormal focus

### ISSUE THREAD

**Issue by bcoleman96** [obophenotype/human-phenotype-ontology#7276]
**Preferred term label:**
Fixated interest with abnormal focus

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
Restricted interests (See issue #7275)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-4422-1708

**Comment by pnrobinson** [comment]
Sorry, this addresses the previous comment ( #7277 ). Please provide the full suggested definition here together with the citation (ISBN if for DSM-V)
